### ISSUE TITLE
508-defect-4 [COGNITION]: Direct Deposit Component - Consider moving the informational note above the direct deposit info/edit in place form

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#18607]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a user, I would like to read or hear relevant information about my account before I enter any data related to direct deposit. I like to hear all the information first, then make a decision.

## Details

I found myself asking if it would make sense to have the noted information and the Additional Info component above the Direct Deposit edit in place form/info card. This is an admittedly subjective review item. If you all feel the current layout is the best one, please feel free to comment and close.

## Acceptance Criteria

- [ ] Review the ticket and either reorder content or comment and close if the team opts not to change the layout
- [ ] If you do change the layout, please consider removing the `tabindex="0"` attribute on the Note... block. Having this text next to the heading and ahead of the form should be enough prominence.

## WCAG or Vendor Guidance (optional)

* [Labels or Instructions: Understanding SC 3.3.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-cues.html)

## Screenshots or Trace Logs

![Screen Shot 2021-01-14 at 3 37 57 PM](https://user-images.githubusercontent.com/934879/104652964-7cd37400-567f-11eb-93a9-d698e9cc18d8.png)
